### PR TITLE
JS Networking Client

### LINKED ISSUE #109: Add API client using axios and API interface
Add a real implementation of the frontend API interface that uses axios to fetch from the backend apis.

### PR DESCRIPTION
**Pull request by timtraversy**
@alexjball 

Closes #109 

This is the generated implementation of the JS client. Need to actually hook it into the React components before merging.

### CODE REVIEW

File: frontend/src/api/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
```
Comment (alexjball) [inline frontend/src/api/.travis.yml:1]: You can remove this file. We are using Github actions. If you're up for setting up CI for these tests, you can create a workflow like [this one](https://github.com/codeforboston/jobhopper/blob/11d547a9c36681fdd94109d13d275513f9764c39/.github%2Fworkflows%2Flint-test.yml)

File: frontend/src/api/git_push.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```
Comment (alexjball) [inline frontend/src/api/git_push.sh:1]: Do we need this file? what is it for?

File: frontend/src/api/src/ApiClient.js [hunk 3]
```diff
@@ -0,0 +1,653 @@
+/**
```
Comment (alexjball) [inline frontend/src/api/src/ApiClient.js:1]: Nice!

**Review by alexjball — COMMENTED** [review]
Looks good! Since this is an independent package, I think it should live in a sibling directory of the frontend, maybe called api-client. We would then use [npm link](https://docs.npmjs.com/cli/v6/commands/npm-link) to add it as a dependency to the frontend.

Separately, we'll need to figure out how to build this for use in the browser.

**Comment by alexjball** [comment]
I opened #237 in the interest of having something for Tuesday

**Comment by jedpittman** [comment]
@alexjball , can I close this PR as well since it should be covered already?

**Comment by alexjball** [comment]
Hey @jedpittman yes we can close this

**Pull request closed without merging**